index.js

```diff
@@ -19,4 +21,120 @@ navLink.forEach((link) => {
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    featuredImages: '',
+    technologies: ['HTML', 'BootStrap', 'Ruby'],
+    linktoliveversion: '',
+    linktosource: '',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    featuredImages: '',
+    technologies: ['HTML', 'BootStrap', 'Ruby'],
+    linktoliveversion: '',
+    linktosource: '',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    featuredImages: '',
+    technologies: ['HTML', 'BootStrap', 'Ruby'],
+    linktoliveversion: '',
+    linktosource: '',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    featuredImages: '',
+    technologies: ['HTML', 'BootStrap', 'Ruby'],
+    linktoliveversion: '',
+    linktosource: '',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    featuredImages: '',
+    technologies: ['HTML', 'BootStrap', 'Ruby'],
+    linktoliveversion: '',
+    linktosource: '',
+  },
+];
+
+// select projects list tag from html file
+const projectsListTag = document.querySelector('.projects-list');
+// const project = { technologies: ['HTML', 'BootStrap', 'Ruby'] };
+projects.forEach((project) => {
+  // create single card html element
+  const projectCard = document.createElement('section');
+  projectCard.classList.add('ex1');
+  projectCard.innerHTML = `
+    <h2 class="header-2">${project.name}</h2>
+    <p class="para-2">${project.description}</p>
+    <ul class="apps-1">
+      <li class="i">${project.technologies[0]}</li>
+      <li class="i">${project.technologies[1]}</li>
+      <li class="i">${project.technologies[2]}</li>
+    </ul>
+    <button class="btn-5">
+      See Project
+    </button>
+  `;
+  projectsListTag.appendChild(projectCard);
+});
+
+// const projectCardPopup = document.createElement('section');
+// projectCardPopup.classList.add('project-card-popup');
+// projectCardPopup.classList.add('hidden');
+// projectCardPopup.innerHTML = `
+//   <h2 class="header-2">${project.name}</h2>
+//   <p class="para-2">${project.description}</p>
+//   <ul class="apps-1">
+//     <li class="i">${project.technologies[0]}</li>
+//     <li class="i">${project.technologies[1]}</li>
+//     <li class="i">${project.technologies[2]}</li>
+//   </ul>
+//   <button class="btn-5">
+//     See Project
+//   </button>
+// `;
+
+// projectsListTag.appendChild(projectCardPopup);
```

Comment:
- [ ] [Optional]🔍 It's important to address the issue of unnecessary commented code in order to improve the quality and readability of your code. Comments that do not add value or relevant information can clutter the structure and make it harder for others to understand and navigate.

🧹 Take the time to thoroughly review your code and remove any comments that don't contribute to the understanding or usefulness of the code. This will ensure that your codebase remains organized and easy to work with.

✂️ By removing unnecessary comments, you'll create a more streamlined codebase that is easier to comprehend and maintain. This will benefit both you and anyone working on the project in the future.